csharp/Api/Concept/Type/IThingType.cs

```diff
@@ -335,7 +335,7 @@ IThingType IConcept.AsThingType()
         public class Annotation : NativeObjectWrapper<Pinvoke.Annotation>
         {
             /**
-             * @hidden
+             * @private
```

Comment:
The equivalent of @hidden for the java documentation. It causes warnings that a member lacks documentation while the doxygen build (which is dumb), but I find it okay.